mips_api/mips.py

```diff
@@ -9,33 +10,50 @@ class App:
     _mips_url_chart = 'https://mipapi.abilaonline.com/api/v1/maintain/chartofaccounts'
     _mips_url_logout = 'https://mipapi.abilaonline.com/api/security/logout'
 
+    # add these meta codes as valid `CostCenter` values
+    _extra_program_codes = {
+        '000000':  'No Program',
+        '000001':  'Other',
+    }
+
+    # ignore these active codes
+    _omit_program_codes = [
+        '999900',  # unfunded
+        '999800',  # salary cap
+        '999700',  # long term leave
+        '990500',  # program management
```

Comment:
it would be better to read this data in from an external source so we can update without updating code

Reply:
are you suggesting we move it outside of a git repo? if we move it to a configuration file, I would still want it tracked in version control, so I don't see any difference in the burden to making updates.